doc/source/community.rst

```diff
@@ -7,30 +7,33 @@ Community
 There are some places for sharing Groonga information.
 We welcome you to join our community.
 
-Mailing List
-------------
+Chat room
+---------
 
-There are mailing lists for discussion about Groonga.
+There is a chat room for discussion about Groonga on Zulip.
 
-For English speakers
-  `groonga-talk@lists.sourceforge.net <http://lists.sourceforge.net/mailman/listinfo/groonga-talk>`_
+https://groonga.zulipchat.com/
 
-For Japanese speakers
-  `groonga-dev@lists.osdn.me <http://lists.osdn.me/mailman/listinfo/groonga-dev>`_
+Thanks
+^^^^^^
 
-Chat room
----------
+We're using `Zulip's sponsorship for open source projects <https://zulip.com/for/open-source/>`_.
+
+|inline-zulip-logo| [#zulip-license]_ `Zulip <https://zulip.com/>`_ is an organized team chat app designed for efficient communication.
 
-There are chat rooms for discussion about Groonga.
+.. |inline-zulip-logo| image:: /images/zulip-icon-128x128.png
+   :height: 2em
+   :target: https://zulip.com/
 
-For English speakers
-  `groonga/en chat room on Gitter <https://gitter.im/groonga/en>`_
+.. [#zulip-license]
+   Copyright 2011-2020 Dropbox, Inc., Kandra Labs, Inc., and contributors
+   (`NOTICE <https://github.com/zulip/zulip/blob/c1c9a2e8ee9373fb224764a2bce46f6c8f92225e/NOTICE>`_\)
```

Comment:
Why do we need `\` before `)`?

`_ )` may be better than `_\)`.

Reply:
It was ok without escaping, so I removed it.
(I got an error during the update, but I think it was caused by something different.)